packages/components/components/elvis-accordion/package.json

```diff
@@ -19,7 +19,7 @@
     "@elvia/elvis-assets-icons": "^3.10.0",
     "@elvia/elvis-colors": "^4.4.1",
     "@elvia/elvis-component-wrapper": "^4.2.0",
-    "@elvia/elvis-toolbox": "^12.0.1",
+    "@elvia/elvis-toolbox": "^12.1.0",
```

Comment:
You can also update the dependency on `@elvia/elvis-typography` to the newest on all the components you do this to! 😄